### DISCUSSION TITLE
I think I use a lot of storage resources of Pockethost.

### DISCUSSION THREAD

**Discussion by tekojtrapkomo** [pockethost/pockethost#178] [Q&A] [answered]
Hello people!
Are there any storage caps in Pockethost? I legit don't want to flood the whole storage because of my app's users can upload files and stuff. 

I'd gladly migrate to VPS or something if this is a issue.

**Comment by benallfree** [comment, marked as answer]
For now this seems like a good use case as PocketHost is developed. Please keep using the platform!

As (if) we approach v1, we may impose resource limitations and paid plans. More on that as things get closer to v1.

**Reply by tekojtrapkomo** [reply]
Gotcha! Thank you for the amazing platform
